src/py21cmmc/mcmc.py

```diff
@@ -180,8 +180,144 @@ def run_mcmc(
         ),
         **mcmc_options,
     )
-
+    
     # The sampler writes to file, so no need to save anything ourselves.
     sampler.startSampling()
+    
+    return sampler
+
+
+def run_multinest(
+    core_modules,
+    likelihood_modules,
+    params,
+    datadir=".",
+    model_name="21CMMC",
+    log_level_21CMMC=None,
+    n_live_points=100,
+    importance_nested_sampling=True,
+    sampling_efficiency=0.8,
+    evidence_tolerance=0.5,
+    max_iter=50,
+    multimodal=True,
+    write_output=True,
+    resume=True,
+    **multinest_options,
```

Comment:
The "Essential" multinest options should be put in here explicitly, and the rest can be caught by the ``**multinest_options`` here.